src/main/scala/beam/router/BeamRouter.scala

```diff
@@ -486,7 +490,29 @@ object BeamRouter {
     transitVehicles: Vehicles,
     fareCalculator: FareCalculator,
     tollCalculator: TollCalculator
-  ) =
+  ) = {
```

Comment:
Reading this makes me believe that this check needs done in the ClusterWorkerRouter also? Or the overloaded constructor of R5RoutingWorker (which is what the ClusterWorkerRouter uses)?

Reply:
No, ClusterWorkerRouter is basically the thing that starts the R5 on other machines.